package/tests/test_server.py

```diff
@@ -278,50 +274,48 @@ def test_pipeline_flag_non_existent(cli_runner):
     assert "Failed to find the pipeline." in result.output
 
 
-@mock.patch("kedro.__version__", "0.15.0")
-def test_viz_kedro15(cli_runner):
+def test_viz_kedro15(mocker, cli_runner):
     """Test that running viz in Kedro 0.15.0."""
+    mocker.patch("kedro.__version__", "0.15.0")
```

Comment:
Is that equivalent to below?
```suggestion
    mocker.patch("kedro.__version__", return_value="0.15.0")
```
If yes I'd prefer we specify it by keyword args to make it clearer. (applies to cases below too)

Reply:
`return_value` is needed when you mock a callable which you want to return a particular value. 
Second positional argument will go into `new` argument of `patch`, which will replace the very object instead.